go-controller/pkg/util/node_annotations.go

```diff
@@ -1402,7 +1402,7 @@ func filterIPVersion(cidrs []netip.Prefix, v6 bool) []netip.Prefix {
 // GetNetworkID will retrieve the network id for the specified network from the
 // first node that contains that network at the network id annotations, it will
 // return at the first ocurrence, rest of nodes will not be parsed.
-func GetNetworkID(nodes []*corev1.Node, nInfo NetInfo) (int, error) {
+func GetNetworkID(nodes []*corev1.Node, nInfo BasicNetInfo) (int, error) {
```

Comment:
why switch this?

Reply:
this function is also now invoked with child interface `BasicNetInfo`, so had to change this.